onnx/defs/tensor/defs.cc

```diff
@@ -1325,6 +1325,69 @@ ONNX_OPERATOR_SET_SCHEMA(
           return;
         }));
 
+void resizeShapeInference(InferenceContext& ctx) {
+  if (!hasNInputShapes(ctx, 2)) {
```

Comment:
I think we need to only check for the first input shape. Even if the second input shape is missing, we can still proceed further and atleast trigger rank inference section. What do you think about this ?